lib/guessingGameMakeGuess.js

```diff
@@ -2,7 +2,14 @@
  * Given a game and guess,
  */
 function guessingGameMakeGuess(game, guess) {
-
+  game.numGuesses++;
```

Comment:
@wismith This is good. I'd add some blank lines just to make it easier to read. I'd also keep thematically-similar lines together. For example:

```javascript
function guessingGameMakeGuess(game, guess) {
  let targetWord = game.targetWord;
  let currentLetters = game.currentWord.split('');

  for (let i = 0; i < targetWord.length; i++) {
    if (targetWord[i] === guess) {
      currentLetters[i] = guess;
    }
  }
  
  game.currentWord = currentLetters.join('');
  game.numGuesses += 1;
}
```

"Thematic" sections are separated by some whitespace and all the state-modifying code is right next to each other.